### PR TITLE
[Epic] Prevent issue if platform is undefined in game metadata

### PR DESCRIPTION
**Pull request by arielj**
This should fix https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/2526.

Seems like there must be some game that has `undefined` as the platform info instead of an array, this should prevent that from making it crash.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by arielj** [comment]
@CommandMC looks like the problem are some DLCs, I'm not sure what's that game yet

**Comment by CommandMC** [comment]
https://store.epicgames.com/en-US/p/south-park-the-fractured-but-whole--from-dusk-till-casa-bonita lists the DLC as supporting Windows.
Whether that's derived from the main game item or just a default value, I cannot say.

Defaulting to the platforms the main game supports sounds fine

File: src/backend/legendary/library.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { LegendaryInstallPlatform } from './../../common/types/legendary'
```
Comment (CommandMC) [inline src/backend/legendary/library.ts:1]:
```suggestion
import { LegendaryInstallPlatform } from 'common/types/legendary'
```
You might want to change `JavaScript -> Preferences: Import Module Specifier` to "shortest" (or "non-relative") in your VSCode settings

Comment (arielj) [reply src/backend/legendary/library.ts]: weird, I already had that in `shortest`, I'll check again next time it auto imports something

Comment (CommandMC) [reply src/backend/legendary/library.ts]: I also remember having issues with this back when I was using VSCode. We probably have to explicitly tell it about those import paths *somewhere*

**Review by CommandMC — APPROVED** [review]

**Pull request merged**